src/mlpack/methods/preprocess/preprocess_split_main.cpp

```diff
@@ -69,6 +69,21 @@ BINDING_EXAMPLE(
         "test_ratio", 0.3, "training", "X_train", "training_labels", "y_train",
         "test", "X_test", "test_labels", "y_test"));
 
+BINDING_EXAMPLE(
+    "There is an option to stratify the dataset according to the labels." +
+    PRINT_PARAM_STRING("stratify_data") + " option to stratify the data;"
+    "an example to stratify the data is" +
```

Comment:
These sentences seem really redundant.  Maybe something more like:

"To use stratified sampling so that the ratio of classes in the training and test set match the ratio of classes in the input data, we can specify the " + PRINT_PARAM_STRING("stratify_data") + " option."

(Or something like this.)